docs/ise_compatibility.md

```diff
@@ -0,0 +1,118 @@
+# ISE Compatibility
+
+While the PowerShell extension for VSCode does not seek
+100% compatibility/reproduction of features in the PowerShell ISE,
+there are features in place to make the VSCode experience more natural
+for users of the ISE.
+
+This document tries to list settings you can configure in VSCode
+to make the user experience a bit more familiar compared to the ISE.
+
+## Key bindings
+
+| Function                              | ISE Binding                  | VSCode Binding                              |
+| ----------------                      | -----------                  | --------------                              |
+| Interrupt and break debugger          | <kbd>Ctrl</kbd>+<kbd>B</kbd> | <kbd>F6</kbd>                               |
+| Execute current line/highlighted text | <kbd>F8</kbd>                | <kbd>F8</kbd>                               |
+| List available snippets               |                              | <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>J</kbd> |
+
+## Tab-completion
+
+To enable more ISE-like tab-completion, add this setting:
+
+```json
+"editor.tabCompletion": "on"
+```
+
+This is a setting added directly to VSCode (rather than in the extension),
+so its behavior is determined by VSCode directly and cannot be changed by the extension.
+
+## No focus on console when executing
+
+To keep the focus in the editor when you execute with <kbd>F8</kbd>:
+
+```json
+"powershell.integratedConsole.focusConsoleOnExecute": false
+```
+
+The default is `true` for accessibility purposes.
+
+## Do not start integrated console on startup
+
+To stop the integrated console on startup, set:
+
+```json
+"powershell.integratedConsole.showOnStartup": false
+```
+
+Note that the background PowerShell process will still start,
+since that provides intellisense, script analysis, symbol navigation, etc.
+But the console will not be shown.
+
+## Assume files are PowerShell by default
+
+To make new/untitled files register as PowerShell by default:
+
+```json
+"files.defaultLanguage": "powershell"
+```
+
+## Colorscheme
+
+There are a number of ISE themes available for VSCode
+to make the editor look much more like the ISE.
+
+In the [Command Palette]
+type `theme` to get `Preferences: Color Theme` and press <kbd>Enter</kbd>.
+In the drop down list, select `PowerShell ISE`.
+
+You can set this in the settings with:
+
+```json
+"workbench.colorTheme": "PowerShell ISE"
+```
+
+## Show-Command
```

Comment:
```suggestion
## Command explorer
```
If it doesn't actually replace `Show-Command` this title would be confusing.

Reply:
My intent is for the titles to be keyed to ISE vocabulary as a lookup for new concepts, but I can change